packages/snjs/lib/utils.ts

```diff
@@ -36,7 +36,7 @@ export function isWebCryptoAvailable() {
 /**
  * Whether we are in React Native app
  */
-export function isReactNativeEnvironment() {
+export function isReactNativeEnvironment() : boolean {
```

Comment:
Incorrect spacing, should be using auto-linter. It's option + shift + f on Mac also to perform auto-format.